src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -259,6 +259,17 @@ export const TradeConfirm = ({ history }: RouterProps) => {
                   </Stack>
                 </Alert>
               )}
+              {trade?.buyAsset.assetId === thorchainAssetId && (
+                <Alert status='info' mx={3} width='auto' mb={3} fontSize='sm'>
+                  <AlertIcon />
+                  <Stack spacing={0}>
+                    <AlertTitle>{translate('trade.intoRUNETitle')}</AlertTitle>
```

Comment:
Non-blocking, but can we memoize this? Function calls straight from JSX components make us one more lint error away from a fully-memoized codebase